### PR TITLE
Hotfix/config files

### PR DESCRIPTION
**Pull request by sirgalleto**
# What does this PR do?

Fix the use of config package and use directly the json file to get the config vars.

### CODE REVIEW

**Review by timgivois — APPROVED** [review]

File: cli.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env node
 
 const program = require('commander');
-const config = require('config');
+const config = require('./config/defaultProps.json');
```
Comment (renemichel) [inline cli.js]: isn't the file `default.json`?

**Review by renemichel — CHANGES_REQUESTED** [review]
I think the filename is not correct on the config assignment

**Comment by renemichel** [comment]
image

File: cli.js (hunk 1 above)
Comment (sirgalleto) [reply cli.js]: Oh yes, done!

File: index.js [hunk 2]
```diff
@@ -2,15 +2,15 @@ const _ = require('lodash');
 const path = require('path');
 const notifier = require('node-notifier');
 const formatNum = require('format-num');
-const config = require('config');
 const db = require('./db');
 const getBitsoExchange = require('./services/bitso');
+const defaultProps = require('./config/defaultProps.json');
```
Comment (renemichel) [inline index.js]: here as well bro :p

Comment (sirgalleto) [reply index.js]: Done again haha

**Review by renemichel — APPROVED** [review]
:shipit: LGTM

**Pull request merged**